index.js

```diff
@@ -26,7 +29,14 @@ function dynalite(options) {
 	options = options || {}
 	var server, stores = {}, requestHandler = httpHandler.bind(null, stores, options)
 
-	logger = bunyan.createLogger("dynalite_log", __dirname, options)
+	logger = dynaliteLogger.createLogger("dynalite_log", __dirname, options)
+	
+	if (options.memoryStats) {
+		setInterval(function() {
+			logger.info("Process Memroy Stats: " + stringify(process.memoryUsage()));
```

Comment:
Can `stringify(process.memoryUsage())` fail?

Probably should surround this with try/catch